lib/bundler/plugin.rb

```diff
@@ -45,6 +45,22 @@ def install(names, options)
       Bundler.ui.error "Failed to install plugin #{name}: #{e.message}\n  #{e.backtrace.join("\n ")}"
     end
 
+    # List installed plugins and commands
+    #
+    def list
+      output = ""
```

Comment:
Where is the best place to format the string that will be printed as result of the command? And where this should it actually be printed?